### PR TITLE
Update connector for Ballerina 1.2.0

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
$title

## Remarks
Need to test

### CODE REVIEW

File: cassandra/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[project]
+org-name="ballerina"
```
Comment (daneshk) [inline cassandra/Ballerina.toml:2]:
```suggestion
org-name="ballerinax"
```

File: cassandra/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+version="0.8.5"
+
+[platform]
+target = "java8"
+
+[[platform.libraries]]
+module = "cassandra"
+path = "../cassandra-utils/target/cassandra-utils-0.8.5.jar"
+artafactId = "redis"
```
Comment (daneshk) [inline cassandra/Ballerina.toml:11]:
```suggestion
artafactId = "cassandra-utils"
```

File: cassandra/Ballerina.toml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+version = "0.8.5"
+groupId = "org.ballerinalang"
```
Comment (daneshk) [inline cassandra/Ballerina.toml:13]:
```suggestion
groupId = "org.ballerinalang"

```

File: cassandra-utils/pom.xml [hunk 4]
```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (daneshk) [inline cassandra-utils/pom.xml:3]:
```suggestion
  ~ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: cassandra-utils/src/main/java/org.ballerinalang.cassandra/BCursorTable.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
+package org.ballerinalang.cassandra;
```
Comment (kalaiyarasiganeshalingam) [inline cassandra-utils/src/main/java/org.ballerinalang.cassandra/BCursorTable.java:1]: Can we add the license header?

File: cassandra/src/cassandra/endpoint.bal [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    public remote function 'select(string queryString, typedesc<record {|any|error...;|}> recordType,
+        Param... parameters) returns table<record {}>|error {
+        return externSelect(self, java:fromString(queryString), recordType, parameters);
+    }
+
+    # Execute update query on cassandra datasource.
+    #
+    # + queryString - Query to be executed
+    # + parameters - The parameters to be passed to the update query
+    # + return - `nil` upon success or `error` if an error occurs
```
Comment (kalaiyarasiganeshalingam) [inline cassandra/src/cassandra/endpoint.bal:46]:
```suggestion
    # + return - `()` upon success or `error` if an error occurs
```

**Pull request merged**